janusgraph-core/src/main/java/org/janusgraph/core/JanusGraphFactory.java

```diff
@@ -104,7 +110,56 @@ public static JanusGraph open(BasicConfiguration configuration) {
      * @return JanusGraph graph database
      */
     public static JanusGraph open(ReadConfiguration configuration) {
-        return new StandardJanusGraph(new GraphDatabaseConfiguration(configuration));
+        return open(configuration, null);
+    }
+
+    public static JanusGraph open(ReadConfiguration configuration, String backupName) {
```

Comment:
Need some javadoc especially for backupName parameter.

Reply:
see above.